moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_joints_widget.cpp

```diff
@@ -535,30 +590,37 @@ bool JointsWidgetEventFilter::eventFilter(QObject* /*target*/, QEvent* event)
   else if (event->type() == QEvent::KeyPress)
   {
     QKeyEvent* key_event = static_cast<QKeyEvent*>(event);
-    if (key_event->key() != Qt::Key_Left && key_event->key() != Qt::Key_Right)
-      return false;  // only react to KeyLeft / KeyRight events
+
+    if (key_event->key() != Qt::Key_Left && key_event->key() != Qt::Key_Right && key_event->key() != Qt::Key_Return)
+      return false;  // only react to these events
 
     QAbstractItemView* view = qobject_cast<QAbstractItemView*>(parent());
     QModelIndex index = view->currentIndex();
     index = index.sibling(index.row(), 1);
 
+    if (key_event->key() == Qt::Key_Return)
+    {
+      view->edit(index);
+      return false;
+    }
+
     if (key_event->type() == QEvent::KeyPress && key_event->modifiers() == Qt::NoModifier &&
         index.flags() & Qt::ItemIsEditable)
     {
       if (!key_event->isAutoRepeat())  // first key press: initialize delta_ from joint type and direction
       {
-        delta_ = key_event->key() == Qt::Key_Left ? -0.1f : 0.1f;
+        delta_ = key_event->key() == Qt::Key_Left ? -0.01f : 0.01f;
```

Comment:
In my opinion, the start delta was too large (for revolute joints)
```suggestion
        delta_ = key_event->key() == Qt::Key_Left ? -0.002f : 0.002f;
```